### ISSUE TITLE
Windows Search from Start Bar - Ignore it

### ISSUE THREAD

**Issue by jaystarkey** [digitalcreations/MaxTo#327]
I was playing around with Windows + NumPad and accidentally moved the Windows home search widget around the screen.  Now I can't get it to go back to the correct spot in the lower left hand corner.  Probably should not allow movement of the windows start bar and related functions around the screen -- if there are users who want to do that, maybe make it a buried option somewhere?  Anyhow, I need to get back to the right spot and am not sure how to do so...

**Comment by vegardlarsen** [comment]
This issue will correct itself after a reboot, or after killing `explorer.exe` and restarting it using Task Manager.

I have confirmed that it happens and am working on a fix that will be delivered as a compatibility shim, so it won't require a full update of MaxTo. More details to follow as I have them.

**Comment by vegardlarsen** [comment]
Thank you for this report. It took me a lot longer to figure out than I expected. It is not fixable just by adding a compatibility shim, we needed code changes in MaxTo as well. Those will ship with the next release.

**Comment by jaystarkey** [comment]
Thanks - I have a Samsung CJ890, replacing a 3 monitor setup, and I find the native Windows snapping ability to be less than desirable.  I tried Samsung's app, which was horrible and buggy (not buggy like sometimes can do things like move the search bar around because pesky users like me use it in unexpected ways, buggy like the core code was done in a day and it just isn't very polished).  I can't remember how I stumbled across your app, but I downloaded it, tested it out, and bought it.  Going to enjoy using it -- thanks so much!
